chrome/run.js

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
+const chromeLaunch = require('chrome-launch'); // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
I *don't* think the `no-extraneous-deps` comment is needed since chrome-launch is in our package.json. Does eslint throw an error if you remove it?

(you can test by running `npm run eslint`)